src/openvic-simulation/dataloader/Dataloader.cpp

```diff
@@ -521,9 +566,11 @@ bool Dataloader::_load_map_dir(GameManager& game_manager, fs::path const& map_di
 	std::vector<std::string_view> water_province_identifiers;
 
 #define APPLY_TO_MAP_PATHS(F) \
-	F(definitions) F(provinces) F(positions) F(terrain) F(rivers) \
-	F(terrain_definition) F(tree_definition) F(continent) F(adjacencies) \
-	F(region) F(region_sea) F(province_flag_sprite)
+	F(definitions) \
+	F(provinces) \
+	F(positions) F(terrain) F(rivers) \
+		F(terrain_definition) F(tree_definition) F(continent) F(adjacencies) \
+			F(region) F(region_sea) F(province_flag_sprite)
```

Comment:
If we're gonna format this like that, might as well format them on each line if clang-format will respect that.